src/DatasetHandler.js

```diff
@@ -973,6 +973,11 @@ module.exports = class DatasetHandler {
 
     if (!prefix) {
       prefix = DatasetHandler.getKnownPrefix(namespace);
+      if (prefix) {
+        debug(
+          `Determined vocabulary prefix [${prefix}] from hard-coded list of well known vocabularies.`
+        );
+      }
```

Comment:
The following `if(!prefix)` block could be hoisted up as an `else` clause to this.

Reply:
Yeah, nice one: 7d19d91..f2b0108